longest_common_subsequence/longestCommonSubsequence.java

```diff
@@ -0,0 +1,44 @@
+
+class LongestCommonSubsequence
+{
+	//Function that returns Longest Common Subsequence of two strings
+	//Time Complexity - O( len(str1) * len(str2) ) 
+	//Space Complexity - O( len(str1)*len(str2) )
+	private static String lcs(String str1, String str2)
+	{
+		int [][] arr=new int[str1.length()+1][str2.length()+1];
```

Comment:
Always introduce space around binary operators and assignment operators- 
```java
int[][] arr = new int[str1.length() + 1][str2.length() + 1];
```
Please make this change in all instances.